commands/sync.go

```diff
@@ -58,21 +81,27 @@ var SyncStatusCmd = &cli.Command{
 			} else {
 				heightDiff = 0
 			}
-			fmt.Printf("\tBase:\t%s\n", base)
-			fmt.Printf("\tTarget:\t%s (%d)\n", target, theight)
-			fmt.Printf("\tHeight diff:\t%d\n", heightDiff)
-			fmt.Printf("\tStage: %s\n", ss.Stage)
-			fmt.Printf("\tHeight: %d\n", ss.Height)
-			if ss.End.IsZero() {
-				if !ss.Start.IsZero() {
-					fmt.Printf("\tElapsed: %s\n", time.Since(ss.Start))
+
+			if !short {
+				fmt.Printf("\tBase:\t%s\n", base)
+				fmt.Printf("\tTarget:\t%s (%d)\n", target, theight)
+				fmt.Printf("\tHeight diff:\t%d\n", heightDiff)
+				fmt.Printf("\tStage: %s\n", ss.Stage)
+				fmt.Printf("\tHeight: %d\n", ss.Height)
+				if ss.End.IsZero() {
+					if !ss.Start.IsZero() {
+						fmt.Printf("\tElapsed: %s\n", time.Since(ss.Start))
+					}
+				} else {
+					fmt.Printf("\tElapsed: %s\n", ss.End.Sub(ss.Start))
+				}
+				if ss.Stage == api.StageSyncErrored {
+					fmt.Printf("\tError: %s\n", ss.Message)
```

Comment:
I think the short output should indicate whether there is an error or not. Not relevant if you choose to output json instead